src/lightning/pytorch/loops/epoch/training_epoch_loop.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, min_steps: Optional[int] = None, max_steps: int = -1) -> None
         self.batch_progress = BatchProgress()
         self.scheduler_progress = SchedulerProgress()
 
-        self.optimizer_loop = _OptimizerLoop()
+        self.optimizer_loop = _AutomaticOptimization()
```

Comment:
nit: can we also rename these variables then?

Reply:
they are public :/

Reply:
Thanks for spotting this. It was incorrectly undone when I was resolving merge conflicts.